.circleci/config.yml

```diff
@@ -335,7 +335,128 @@ jobs:
+                
+                MESSAGE="$MESSAGE\n$CHANGED_PAGES"
+                
+                npx auto comment --message "$(printf "$MESSAGE")" --context "build-preview"
+              else
+                echo "Non-PR branch - docs built, deployment handled by docs_release job"
+              fi
+            else
+              echo "Skipping docs build - not requested"
+              exit 0
+            fi
+
+  docs_release:
+    executor: base
+    steps:
+      - attach_workspace:
+          at: ~/player
+      - restore_cache:
+          keys:
+            - "v1-bazel-cache-core-{{ .Branch }}-{{ .Revision }}"
+            - "v1-bazel-cache-core-{{ .Branch }}"
+            - "v1-bazel-cache-core-main"
+
+      - add_ssh_keys:
+          fingerprints:
+            - "SHA256:hUAWVYdfSW5ml6ufbfyEHUcuPwUJ88xVhsZc21b2lA8"
+
+      - run: mkdir -p ~/.ssh/ && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config
+      - run: git config --global user.name "Intuit Service"
+      - run: git config --global user.email "opensource-svc@intuit.com"
+
+      - run: |
+          echo "Building and deploying docs for release"
+
+          # Build the docs first
+          bazel build --config=release //docs:gh_deploy
+
+          # If GHA_Action = "release" + main branch -> full release (deploy to latest/)
+          # If GHA_Action = "" + main branch -> next release (deploy to next/)
+          if [ "<< pipeline.parameters.GHA_Action >>" = "release" ] && [ "$CIRCLE_BRANCH" = "main" ]; then
+            echo "Full release detected - deploying to latest/"
+            STABLE_DOCS_BASE_PATH="latest" \
+            STABLE_ALGOLIA_SEARCH_API_KEY=$ALGOLIA_SEARCH_API_KEY \
+            STABLE_ALGOLIA_SEARCH_APPID="OX3UZKXCOH" \
+            STABLE_ALGOLIA_SEARCH_INDEX="player-ui" \
+            bazel run --config=release //docs:gh_deploy -- --dest_dir "latest"
+            
+            echo "Docs deployed to: https://player-ui.github.io/latest/"
+          elif [ "<< pipeline.parameters.GHA_Action >>" = "" ] && [ "$CIRCLE_BRANCH" = "main" ]; then
+            echo "Next release detected - deploying to next/"
+            STABLE_DOCS_BASE_PATH="next" \
+            STABLE_ALGOLIA_SEARCH_API_KEY="$ALGOLIA_NEXT_SEARCH_API_KEY" \
+            STABLE_ALGOLIA_SEARCH_APPID="D477I7TDXB" \
+            STABLE_ALGOLIA_SEARCH_INDEX="crawler_Player (Next)" \
+            bazel run --config=release //docs:gh_deploy -- --dest_dir "next"
+            
+            echo "Docs deployed to: https://player-ui.github.io/next/"
+          else
+            echo "No docs deployment needed for this context"
+          fi
+
+          # Also deploy to the versioned folder for releases
+          if [ "<< pipeline.parameters.GHA_Action >>" = "release" ]; then
+            SEMVER_MAJOR=$(cat VERSION | cut -d. -f1)
```

Comment:
So since this isn't orchestrated though auto will the VERSION file have the correct version? Since this stage is not executed through auto I'm not sure it will